windows_bat/PokemonGo-Bot-Install.bat

```diff
@@ -203,14 +201,9 @@ if exist %PGBotPath%\encrypt. copy %PGBotPath%\encrypt. %DownPath%
 if exist %PGBotPath% rmdir %PGBotPath% /s /q
 if not exist %PGBotPath% md %PGBotPath%
 cd C:\Python27\
-pip2 install --upgrade virtualenv

```

Comment:
If I leave virtualenv in the install and start batch, they will just stop in 3 out of 4 Windows machines, show a `Installing pip, setuptools` message and stay there for hours, not showing any error messages.